protos/sigstore_bundle.proto

```diff
@@ -44,34 +44,24 @@ message TimestampVerificationData {
         repeated dev.sigstore.common.v1.RFC3161SignedTimestamp rfc3161_timestamps = 1;
 }
 
-// ExtendedVerificationMaterials contains extra data that can be used to verify things
-// such as transparency and timestamp of the signature creation.
-// As this message can be either empty (no inclusion proof or timestamps), or a combination of
-// an arbitrarily number of transparency log entries and signed timestamps,
-// it is the client's responsibility to implement any required verification
-// policies.
-message ExtendedVerificationMaterials {
-        // This is the inclusion promise and/or proof, where
-        // the timestamp is coming from the transparency log.
-        repeated dev.sigstore.rekor.v1.TransparencyLogEntry tlog_entries = 1;
-        // Timestamp verification data, over the artifact's signature.
-        TimestampVerificationData timestamp_verification_data = 2;
-}
-
 message Bundle {
         // MUST be application/vnd.dev.sigstore.bundle+json;version=0.1
         // when encoded as JSON.
         string media_type = 1;
-        ExtendedVerificationMaterials extended_verification_materials = 2;
-        dev.sigstore.common.v1.VerificationMaterial verification_material = 3;
+        // This is the inclusion promise and/or proof, where
+        // the timestamp is coming from the transparency log.
+        repeated dev.sigstore.rekor.v1.TransparencyLogEntry tlog_entries = 2;
+        // Timestamp verification data, over the artifact's signature.
+        TimestampVerificationData timestamp_verification_data = 3;
+        dev.sigstore.common.v1.VerificationMaterial verification_material = 4;
         oneof content {
-                dev.sigstore.common.v1.MessageSignature message_signature = 4;
+                dev.sigstore.common.v1.MessageSignature message_signature = 5;
```

Comment:
I think the bundle format is now being used by sigstore-js, so we can't renumber the tags (@kommendorkapten ?)